packages/template-retail-react-app/app/pages/checkout/confirmation.test.js

```diff
@@ -9,213 +9,205 @@
 import React from 'react'
 import {screen, waitFor} from '@testing-library/react'
 import user from '@testing-library/user-event'
+import {Route, Switch} from 'react-router-dom'
 import {rest} from 'msw'
 import {renderWithProviders, createPathWithDefaults} from '../../utils/test-utils'
 import Confirmation from './confirmation'
-import {keysToCamel} from '../../commerce-api/utils'
-import useBasket from '../../commerce-api/hooks/useBasket'
-import useShopper from '../../commerce-api/hooks/useShopper'
-import {ocapiOrderResponse} from '../../commerce-api/mock-data'
-import {mockedGuestCustomer, exampleTokenReponse} from '../../commerce-api/mock-data'
 
-jest.mock('../../commerce-api/auth', () => {
-    return class AuthMock {
-        login() {
-            return mockedGuestCustomer
-        }
-    }
-})
-
-jest.mock('commerce-sdk-react-preview', () => {
-    const originalModule = jest.requireActual('commerce-sdk-react-preview')
-    return {
-        ...originalModule,
-        useCustomerBaskets: jest.fn().mockReturnValue({data: {baskets: [{currency: 'GBP'}]}})
-    }
-})
-
-const mockOrder = keysToCamel({
-    basket_id: 'testorderbasket',
-    ...ocapiOrderResponse
-})
-
-const mockBasketOrder = {
-    baskets: [mockOrder]
+const MockedComponent = () => {
+    return (
+        <Switch>
+            <Route path={createPathWithDefaults('/checkout/confirmation/:orderNo')}>
+                <Confirmation />
+            </Route>
+        </Switch>
+    )
 }
 
-const mockProducts = {
-    data: [
+const mockOrder = {
```

Comment:
[nit] Can this mock data live elsewhere to keep the test file smaller? It's more than half the lines of the file!